lib/media/text_engine.js

```diff
@@ -160,6 +160,11 @@ shaka.media.TextEngine.prototype.appendBuffer =
 shaka.media.TextEngine.prototype.remove = function(start, end) {
   // Start the operation asynchronously to avoid blocking the caller.
   return Promise.resolve().then(function() {
+    if (!this.track_.cues) {
+      // This TextTrack was disabled before we started work. When TextTracks
+      // move to disabled state their cues property is set to null
+      return
```

Comment:
This doesn't seem like the correct fix to me.  It seems that we should still remove cues.  Otherwise, when CC is re-enabled, the old cues are still there.  This could cause problems when we switch CC tracks and start adding new cues that conflict with the old ones.